.github/workflows/image_build.yml

```diff
@@ -3,6 +3,8 @@ name: Build KubeAGI arcadia image
 on:
   push:
```

Comment:
Should we remove this line ?  https://github.com/kubeagi/arcadia/blob/main/.github/workflows/image_build.yml#L57

Reply:
Let me try it and only release `v0.1.0` tag when it is released.